common/k8sutils/k8sutils.go

```diff
@@ -3,13 +3,13 @@ package k8sutils
 import (
 	"context"
 	"fmt"
+	"github.com/dell/gofsutil"
```

Comment:
hope this has been tested and found no issues.

Reply:
It has been tested and no issues were found.